src/MusicalScore/Graphical/MusicSheetDrawer.ts

```diff
@@ -417,6 +418,34 @@ export abstract class MusicSheetDrawer {
                 this.drawLabel(label, <number>GraphicalLayers.Notes);
             }
         }
+        // Draw bounding boxes for debug purposes. This has to be at the end because only
+        // then all the calculations and recalculations are done
+        if (process.env.DRAW_BOUNDING_BOX_ELEMENT !== undefined) {
```

Comment:
Does this work? If `process.env.DRAW_BOUNDING_BOX_ELEMENT` is `false` as in the default configuration below, will this evaluate as intended? Webpack docs says you need to put `null` for an env variable to appear undefined.

Reply:
It actual works because false !== undefined is not true BUT the type check for object === false is not. So the funtion is called but does not render.
I will change it to null just to be sure